sshcommand

```diff
@@ -136,6 +136,9 @@ sshcommand-acl-add() {
     KEY=$(cat "$KEY_FILE")
   fi
 
+  local count="$(wc -l <<< "$KEY")"
+  [ "$count" -eq 1 ] || log-fail "Too many keys provided, set one per invocation of sshcommand acl-add <USER> <NAME>"
```

Comment:
Per [bash style guidelines](https://github.com/progrium/bashstyle), please use `[[` instead of `[`